mifospay/src/main/AndroidManifest.xml

```diff
@@ -56,8 +55,8 @@
             android:theme="@style/AppTheme" />
         <activity
             android:name=".receipt.ui.ReceiptActivity"
-            android:theme="@style/AppTheme"
-            android:launchMode="singleTask">
+            android:launchMode="singleTask"
+            android:theme="@style/AppTheme">
```

Comment:
was this rearranging necessary of lines?

Reply:
No, I didn't do it manually though. I guess it happened. But this isnt an issue right? I'll make sure it doesn't happen in my other prs